### PR TITLE
Allow user to specify ICEs to run + add two new ones

### PR DESCRIPTION
**Pull request by BGR360**
Was just going to add the two new ICEs, but some of the existing ICEs take an extraordinary amount of time to run on my machine.

So I just added the ability to optionally filter down to one or more specific ICEs when doing `cargo run`.

cc https://github.com/rust-lang/rust/issues/101696 for the two new ICEs

### CODE REVIEW

**Comment by Alexendoo** [comment]
Nice thanks, will take a look in a moment

Do you know which ones were taking ages on your machine? We try not to have ones that take an inordinate amount of time

**Comment by matthiaskrgr** [comment]
Are they related to save analysis? https://github.com/rust-lang/rust/issues/101505

**Comment by BGR360** [comment]
I don't know how to tell which ones were taking long. They only show up as `....` in the terminal

**Comment by matthiaskrgr** [comment]
Hmm, on my machine a `cargo run` finishes within 42 seconds.
if we hadn't named all the outputs in the rustdoc tests "out.rs" we could tell what file was being checked via `ps -aux` :/

**Comment by BGR360** [comment]
I'm running on an M1 Pro macbook if that makes any difference 🤷 

I'll stick some debug logs in there to figure out which ones are running slow

File: README.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ Contributing to Glacier is fairly easy:
 
 1. Check out [this list][ices] of bugs on the Rust issue tracker.
 2. Pick one.
-3. Create a file in `ices/` with the same digit as the bug.
+3. Create a file in `ices/` with the same number as the issue reporting the ICE.
 4. Copy the code that causes the ICE into your new file.
-5. (optional) Verify it works by running `rustup update nightly`, then `cargo run` to run the tests.
+5. (optional) Verify it works by running `rustup update nightly`, then
+   `cargo run NUMBER` to run your ICE.
```
Comment (JohnTitor) [inline README.md:27]: I guess the below is clearer:

```suggestion
   `cargo run $ISSUE_NUMBER` to run your ICE.
```

**Comment by JohnTitor** [comment]
LGTM! I think we can merge once the comment above is resolved and https://github.com/rust-lang/glacier/pull/1415/commits/09ccc5c1f00a1eed1aef0953be4aa4efe4386d7f is dropped.

**Pull request merged**